src/azure-cli/azure/cli/command_modules/identity/custom.py

```diff
@@ -32,33 +27,3 @@ def list_identity_resources(cmd, resource_group_name, resource_name):
     client = _msi_list_resources_client(cmd.cli_ctx)
     return client.list_associated_resources(resource_group_name=resource_group_name,
                                             resource_name=resource_name)
-
-
-def create_or_update_federated_credential(cmd, client, resource_group_name, identity_name, federated_credential_name,
-                                          issuer=None, subject=None, audiences=None):
-    _default_audiences = ['api://AzureADTokenExchange']
-    audiences = _default_audiences if not audiences else audiences
```

Comment:
This issue #31598 is due to ignoring the default audience logic when migrating Code Gen. Could you submit a PR to resolve this issue? @Sruuujaaan